Tryouts/Visuals/Windows/WindowsVisualUtils/MorganStanley.ComposeUI.Tryouts.Visuals.Windows.WindowsVisualUtils.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
*Windows*VisualUtils - the *Windows* prefix is reduntant - it is already under .Visuals.*Windows*.

Reply:
fixed